core/frontend/src/ContextRealityModelState.ts

```diff
@@ -111,7 +111,7 @@ export async function queryRealityData(criteria: RealityDataQueryCriteria): Prom
 
   const requestContext = await AuthorizedFrontendRequestContext.create();
 
-  const client = new RealityDataClient();
+  const client = new RealityDataAccessClient();

```

Comment:
This will need to be removed. and use the IModelApp instance isntead.

Reply:
There is a test-app ui component that depends on this function. We can remove that component since its functionality is not supported in the interface, or we can move the logic from this function into the ClassificationPanel.ts file.

Reply:
I have removed the above mention ui component and this method.